.github/workflows/inspektor-gadget.yml

```
@@ -205,14 +205,17 @@ jobs:
         IMAGE_TAG=${{ steps.set-repo-determine-image-tag.outputs.image-tag }} \
         make $client
 
+        # We need to append .exe to windows binaries...
+        dot_exe=$(test ${{ matrix.os }} = 'windows' && echo '.exe')
```

I see the CI is failing perhaps if the `test` is failing the jobs exits?
```suggestion
        dot_exe=$(test ${{ matrix.os }} = 'windows' && echo '.exe') || true
```